cluster.tf

```diff
@@ -1,104 +1,71 @@
-##############################################################################
-# Find valid IKS/ROKS Cluster versions for region
-##############################################################################
-
-data "ibm_container_cluster_versions" "cluster_versions" {}
-
-##############################################################################
-
-
 ##############################################################################
 # Cluster Locals
 ##############################################################################
 
 locals {
-  worker_pools_map = module.dynamic_values.worker_pools_map # Convert list to map
-  clusters_map     = module.dynamic_values.clusters_map     # Convert list to map
-  latest_kube_version = {
-    openshift = "${data.ibm_container_cluster_versions.cluster_versions.valid_openshift_versions[length(data.ibm_container_cluster_versions.cluster_versions.valid_openshift_versions) - 1]}_openshift"
-    iks       = data.ibm_container_cluster_versions.cluster_versions.valid_kube_versions[length(data.ibm_container_cluster_versions.cluster_versions.valid_kube_versions) - 1]
-  }
+  clusters_map = module.dynamic_values.clusters_map # Convert list to map
 }
 
 ##############################################################################
-
-
-##############################################################################
-# Create IKS/ROKS on VPC Cluster
+# Create ROKS on VPC Cluster
 ##############################################################################
 
-resource "ibm_container_vpc_cluster" "cluster" {
+
+module "cluster" {
+  depends_on = [
+    module.vpc
+  ]
   for_each          = local.clusters_map
-  name              = "${var.prefix}-${each.value.name}"
-  vpc_id            = each.value.vpc_id
+  source            = "git::https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc.git?ref=v2.3.0"
```

Comment:
Need to have disable_public_service_endpoint set to "true" by default and ability to pass in